pkg/controller/windowsmachine/nodeconfig/nodeconfig.go

```diff
@@ -131,9 +134,8 @@ func (nc *nodeConfig) Configure() error {
 	if err := nc.setNode(); err != nil {
 		return errors.Wrapf(err, "error getting node object for VM %s", nc.ID())
 	}
-	// Apply worker labels
-	if err := nc.applyWorkerLabel(); err != nil {
-		return errors.Wrap(err, "failed applying worker label")
+	if err := nc.applyLabelsAndAnnotations(); err != nil {
```

Comment:
Please have separate functions for labels and annotations. The `Update` can be moved out. This will give us more flexibility in the future.